frontend/src/styles/components.css

```
@@ -497,14 +492,14 @@ button.danger:hover {
 .settings-savebar.dirty {
     box-shadow: 0 -4px 12px rgba(0, 0, 0, 0.08);
-    border-top-color: #e0e0e0;
+    border-top-color: var(--cudly-border);
 }
 
 .settings-savebar.dirty::before {
     content: 'Unsaved changes';
     display: inline-block;
     padding: 0.25rem 0.6rem;
-    background: #fff3cd;
+    background: var(--cudly-surface)3cd;
```

Stylelint (17.11.0)

[error] Unknown unit "cd" (unit-no-unknown)

(unit-no-unknown)